components/list/list-item-mixin.js

```diff
@@ -32,6 +32,7 @@ export const ListItemMixin = superclass => class extends ListItemDragDropMixin(L
 			 * Breakpoints for responsiveness in pixels. There are four different breakpoints and only the four largest breakpoints will be used.
 			 */
 			breakpoints: { type: Array },
```

Comment:
Comment in here so it'll eventually show up in the auto-generate catalogue stuff?